javascript/javascript1/week4/script.js

```diff
@@ -0,0 +1,68 @@
+let user ={
+    userName: "",
+    todo : []
+};
+function getReply(voice) {
+command=voice.toLowerCase();
+if (command.includes("hello my name is")) {
+    let splitedstring=command.split(' ')
+   user.userName=splitedstring[splitedstring.length-1];
+    return`nice to meet you ${user.userName}`  
+
+}else if ('what is my name' == command) {
+    return `Your name is ${user.userName}`
+    
+}else if(command.startsWith("add") && command.endsWith("to my todo")){
+    let todoItem=command.split(' ');
+    let sliceElements=todoItem.slice(1,todoItem.length-3).join(' ');
+    user.todo.push(sliceElements);
+    return `${sliceElements} added to your todo`
+
+}else if(command.startsWith("remove") && command.endsWith("from my todo")){
+    let todoItem=command.split(' ');
+    let sliceElements=(todoItem.slice(1,todoItem.length-3)).join(' ');
+    let index = user.todo.indexOf(sliceElements);
+    if(index > -1){
+       let itemToDelete = user.todo.splice(index,1);
+       return `removed ${itemToDelete} from your todo`       
+    }   
+}else if ('what is on my todo?'== command) {
+    let numberOfItemsOnTodo=user.todo.length;
+    return `your to have ${numberOfItemsOnTodo} todos - ${user.todo}`
+
+}else if ('what day is it today?' == command) {
+    const monthNames = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
```

Comment:
It looks like Javascript can now get the full month name without your own `monthNames` array. See here. https://stackoverflow.com/questions/1643320/get-month-name-from-date/18648314#18648314

Reply:
yeah.., thanks for the simple way.